web-components/src/global.css.js

```diff
@@ -242,6 +251,43 @@ merch-card [slot='heading-xl'] {
     color: var(--merch-color-grey-80);
 }
 
+merch-card [slot='callout-text'] {
+    display: inline-block;
+    margin: var(--consonant-merch-spacing-xxxs) 0px;
+}
+
+.callout-content-wrapper {
+    position: relative;
+    display: inline-grid;
+    background: rgba(203 203 203 / 50%);
+    border-radius: var(--consonant-merch-spacing-xxxs);
+    padding: var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxs);
+}
+
+.callout-content-wrapper-with-icon {
```

Comment:
I don't see this style being used in a test, is it still needed?

Reply:
Yes, it's needed to differentiate padding for callouts with and without icon.

Reply:
can you have at least one usage of it in the test pages?

Reply:
added